src/types.ts

```diff
@@ -32,8 +33,20 @@ export interface ZAuctionInstances {
   [registrarAddress: string]: zAuction.Instance;
 }
 
-export interface RouteUriToInstance {
-  [key: string]: ZAuctionInstances;
+export interface TokenAddressMapping {
+  [address: string]: Mapping;
+}
+
+export interface Mapping {
```

Comment:
What is this a `Mapping` of? This is too generic of a name. It has a specific usage and the name should be more specific to that usage.